datastore/tasks.rb

```diff
@@ -48,14 +49,12 @@ def mark_done task_id
 
 # [START retrieve_entities]
 def list_tasks
-  query = @datastore.query("Task").
-          order("created")
-
+  query = @datastore.query("Task").order("created")
```

Comment:
Frank had the same suggestion on the original PR. We left it as is, my original comment:

> For the datastore queries, it seems like a pattern to put every .(something) on a new line
> https://cloud.google.com/datastore/docs/concepts/queries#datastore-basic-query-ruby
> there are more examples with 1 or 2 where, order, etc. on new lines

I'm ok either way, just wanted to bring it to your attention.